README.md

```diff
@@ -21,6 +21,20 @@ provided by the bot. You will only need to do this once across all repos using o
 
 Please review our [contribution guide](CONTRIBUTING.md).
 
+### Development on Windows
+
+This repository contains symbolic links which don't work properly on Windows. To build this project on Windows, you can use our provided Git aliases to replace symbolic links with junctions.
+
+```sh
+$ git config --local include.path ../dev/.gitconfig
+$ git replace-symlinks # replace symlinks with junctions
+$ git restore-symlinks # restore symlinks
+```
+
+### Using IntelliJ
+
+You can use the built-in sbt shell in IntelliJ without any problems. However, the built-in "Build Project" command may not work. To fix the issue, go to Project Structure -> Project Settings -> Modules, remove the "root" module, and mark `src/main/scala` as Sources for the "spark2_4" and "spark3_0" modules, as well as `src/main/scala-spark2` and `src/main/scala-spark3` directories for corresponding modules.
```

Comment:
I also had to do the following to get these compiled in IntelliJ from a fresh clone:
* Change from `Zinc` to `IDEA` under `Build,Execution,Deployment` -> `Compiler` -> `Scala Compiler` ->` Incrementality type`.
* remove the `hyperspace-sources` module

Can you also mention marking `src/test/scala` (and others) as `Tests` so that you can run them easily inside IntelliJ?

Reply:
One more thing is that `BuildInfo` plugin doesn't seem to run if I build it from IDE, so I had to compile from the CLI `sbt` first, then build it from IDE. This could be an existing behavior, but we may want to call it out while we are at it.

Reply:
I didn't have change that, maybe it's different by OS. I'll update the instruction.

Reply:
BuildInfo was already not working in IntelliJ before this PR. Anyway, let's mention it too.